ngraph/python/tests/test_onnx/test_ops_reduction.py

```diff
@@ -17,7 +17,33 @@
 import onnx
 import pytest
 
-from tests.test_onnx.utils import run_node
+from tests.test_onnx.utils import run_node, unstrict_xfail_issue_35925, strict_xfail_issue_35925
+
+reduce_max = "ReduceMax"
```

Comment:
If these constants are used only once in the entire codebase, let's remove them and use strings directly instead.

Reply:
OK, fixed